internal/api/handlers/requestadapters/schedulers.go

```diff
@@ -339,7 +402,7 @@ func fromApiContainerEnvironments(apiEnvironments []*api.ContainerEnvironment) [
 }
 
 func fromApiForwarders(apiForwarders []*api.Forwarder) []*forwarder.Forwarder {
-	var forwarders []*forwarder.Forwarder
+	forwarders := make([]*forwarder.Forwarder, 0)
```

Comment:
Refactor: This was leading to failing when trying to patch forwarders list to empty list